flink-libraries/flink-gelly-scala/src/main/scala/org/apache/flink/graph/scala/Graph.scala

```diff
@@ -407,6 +407,36 @@ TypeInformation : ClassTag](jgraph: jg.Graph[K, VV, EV]) {
   }
 
   /**
+   * Relabels vertices and edges using the given MapFunction.
+   *
+   * @param translator implements conversion from K to NEW
+   * @return relabeled graph
+   */
+  def translateGraphLabels[NEW: TypeInformation : ClassTag](translator: MapFunction[K, NEW]): Graph[NEW,VV,EV] = {
```

Comment:
In the the Gelly code and docs, we refer to the vertex and edge keys as "Ids", not labels. I think we should rename the methods and javadocs to be consistent.